tests/installation/welcome.pm

```diff
@@ -95,14 +95,15 @@ sub run {
         # In upgrade mode, there is no product list shown in welcome screen
         unless (check_var('ARCH', 's390x') || get_var('UPGRADE')) {
             assert_screen('select-product');
-            my %hotkey = (
-                sles     => 's',
-                sled     => 'u',
-                sles4sap => get_var('OFW') ? 'u' : 'i',
-                hpc      => check_var('ARCH', 'x86_64') ? 'x' : 'u'
-            );
+            # my %hotkey = (
```

Comment:
please don't do this, if need to remove something just delete it, we will have git history to revive it in case we will need it